doc/spec.adoc

```diff
@@ -69,37 +95,58 @@ A channel does not provide any default streams.
 
 ## Channel Schema
 
-A JSON Schema is provided to validate that channels comply with the model described above.
+JSON Schemas are provided to validate that channels and manifests comply with the model described above.
 
 ## Channel Representation
 
-A channel is specified in the YAML language with a link:../core/src/main/resources/org/wildfly/channel/v1.0.0./schema.json[corresponding JSON schema] to validate its structure.
+A channel is specified in the YAML language with a link:../core/src/main/resources/org/wildfly/channel/v2.0.0./schema.json[corresponding JSON schema] to validate its structure.
+
+A manifest is specified in the YAML language with a link:../core/src/main/resources/org/wildfly/manifest/v1.0.0./schema.json[corresponding JSON schema] to validate its structure.
+
+A blocklist is specified in the YAML language with a link:.
+./core/src/main/resources/org/wildfly/blocklist/v1.0.0./schema.json[corresponding JSON schema] to validate its structure.
 
 ### Channel Actions and Responsibilities
 
 #### Create a channel
 
-A Channel is a single file that complies with the channel YAML representation.
-Creating a channel corresponds to the creation of such a file.
+A minimal Channel definition is a single file that complies with the channel YAML representation. The Channel definition may reference two additional resources: a Manifest file and a Blocklist file.
+
+Creating a channel corresponds to the creation of the above files.
 
 #### Publish a channel
-A channel must be “published” so that it can be read (or downloaded) by WildFly provisioning tooling. 
-Channels are published as a Maven artifact with the the classifier `channel` and extension/type `yaml`.
+A channel may be “published” so that it can be read (or downloaded) by WildFly provisioning tooling.
+Channels are published as a Maven artifact with the classifier `channel` and extension/type `yaml`.
+
+Manifest and Blocklist files must be "published" in one of the repositories defined in the Channel so that the Channel is able to resolve them.
```

Comment:
This is no longer accurate as manifest and blocklist can also be referenced as `url` outside of Maven ecosystem.